StreamingKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "StreamingKit"
-  s.version      = "0.1.30"
+  s.version      = "0.1.31"
```

Comment:
@haritowa please remove this unnecessary change from PR (I'm using [GitUp](https://github.com/git-up/GitUp) to edit commits, feel free to ask how to do it).